.travis/install.sh

```diff
@@ -3,54 +3,73 @@
 set -e
 set -x
 
-if [[ "$(uname -s)" == 'Darwin' ]]; then
-    export HOMEBREW_NO_AUTO_UPDATE=1
-    export HOMEBREW_LOGS=~/brew-logs
-    export HOMEBREW_TEMP=~/brew-temp
-    mkdir $HOMEBREW_LOGS
-    mkdir $HOMEBREW_TEMP
-    brew update > /dev/null
-    brew cleanup > /dev/null  # do this here, so it won't automatically trigger in the middle of other stuff
-    brew outdated pkg-config || brew upgrade pkg-config
-    # do NOT update openssl 1.0.x, brew will also update a lot of dependent pkgs (and their dependencies) then!
-    #brew outdated openssl || brew upgrade openssl
-    export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig:$PKG_CONFIG_PATH"
-    brew install readline
-    export PKG_CONFIG_PATH="/usr/local/opt/readline/lib/pkgconfig:$PKG_CONFIG_PATH"
-    brew install zstd
-    brew install lz4
-    brew install xz  # required for python lzma module
-    brew install Caskroom/cask/osxfuse
-
-    brew outdated pyenv || brew upgrade pyenv
-    if which pyenv > /dev/null; then
-        eval "$(pyenv init -)"
-    fi
+# Travis clones with depth=50 which might be too shallow for git describe, see https://github.com/pypa/setuptools_scm/issues/93
+git fetch --unshallow --tags
+
+if [ "${TRAVIS_OS_NAME}" = "osx" ]
+then
+
+    # Update brew itself
+    export HOMEBREW_NO_AUTO_UPDATE=1  # Auto-updating everything would take too much time
+    brew update
+    brew cleanup  # Preempt possible scheduled clean-up so it doesn't clutter the log later
+
+    # Install and/or upgrade dependencies
+    #brew install zstd || brew upgrade zstd  # Installation takes very long due to CMake dependency and isn't necessary for the tests as borg comes bundled with zstd anyway
+    brew install lz4 || brew upgrade lz4
+    brew install xz || brew upgrade xz  # Required for Python lzma module
+    brew install Caskroom/cask/osxfuse || brew upgrade Caskroom/cask/osxfuse  # Required for Python llfuse module
+    brew install pyenv || brew upgrade pyenv
```

Comment:
@ThomasWaldmann Or would you rather use `brew bundle` at this point or `for FORMULA in lz4 xz Caskroom/cask/osxfuse pyenv`?

Reply:
i've no idea about brew, not being a macOS user except borg testing.

Reply:
I think there are three possibilities:

- `brew install lz4 || brew upgrade lz4` for each formula (see current code, introduced with 25d3ff783b9d989dcc495ba7212d20d0c803056e)
- `for FORMULA in lz4 xz Caskroom/cask/osxfuse pyenv` with `brew install $FORMULA || brew upgrade $FORMULA`
- `brew bundle` which is like `pip -r` where requirements.txt is called Brewfile to specify formulae but can also be read from stdin

Reply:
But that's just a question of code style. The current code now works again for xcode11.3 https://travis-ci.org/github/borgbackup/borg/builds/666803536

Reply:
for now keep the current style, more tuning can be done in a future PR. and I guess I prefer having it in-file, not in an external file in this case.

Reply:
I agree. :+1: